sdk/core/azure-core-http-netty/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added an override method `sendSync(HttpRequest request, Context context)` to remove reactor exceptions. 
```

Comment:
Since the changes are not public, we don't need to include the method signature here. Also, we should not mention "to remove reactor excpetions". We should just mention that we now support synchronous calls in Netty client.